lib/LedgerSMB/Scripts/payment.pm

```diff
@@ -416,7 +416,7 @@ sub print {
             my $amt = $check->{amount}->copy;
             $amt->bfloor();
             $check->{text_amount} = $payment->text_amount($amt);
-            $check->{decimal} = ($check->{amount} - $amt) * 100;
+            $check->{decimal} = ($check->{amount} - $amt) * 100;  ## no critic (ProhibitMagicNumbers) sniff
```

Comment:
I thought you introduced a "fractional currency to cents" conversion factor? that could be applied here.

Reply:
I thought I got all of these.  I'll get to this.

Reply:
Actually I PRed and you accepted 100 as a non-magical number.

Reply:
Ok. But if that's the case, then we don't need to mark 100 here as a magical number for which the critic needs to be suppressed, right?